Automattic/jetpack
# Adds Backup plugin components / styles for successful backup state

#### Changes proposed in this Pull Request:
* Expands on the Admin Component to add: Backup and StatBlock components
* Adds some general RNA styles
* Adds /jetpack/v4/backups end-point for pulling latest backups info

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:






* Apply the changes to the backup plugin installed on a site
* Ensure the site has a valid backup, and it is the most recent backup.
* The UI should pull Post, Uploads, Themes, and Plugin numbers, along with the date of the relative time of the backup.
* UI should be responsive (changes layout based on width of the screen).  Three break-points are utilized, Roughly these are: desktop, tablet and mobile.


<img width="1587" alt="Screen Shot 2021-06-28 at 6 28 41 PM" src="https://user-images.githubusercontent.com/1992658/123711666-d5b65380-d83e-11eb-9126-617bdfb4e5c5.png">
